docs/_includes/common/filter-menu-button.html

```diff
@@ -0,0 +1,654 @@
+<div id="filter-menu-button">
+    <h2><span class="secondary-text">@ebay/skin/</span>filter-menu-button</h2>
+    <p>Use the <span class="highlight">filter-menu-button</span> class, in conjunction with selected/unselected modifiers,
+        as well as the <a href="#filter-menu">filter-menu</a>, to create a button which opens a menu of options by which to filter.</p>
+
+    <h3 id="filter-menu-button-default">Default Filter Menu Button</h3>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="filter-group">
+                <span class="filter-menu-button">
+                    <button type="button" class="filter-menu-button__button">
+                        <span class="filter-menu-button__button-cell">
+                            <span class="filter-menu-button__button-text">Filter Menu Button</span>
+                            <svg class="icon icon--chevron-down" focusable="false" height="12" width="12">
+                                <use xlink:href="#icon-chevron-down"></use>
+                            </svg>
+                        </span>
+                    </button>
+                    <div class="filter-menu-button__menu">
+                        <div class="filter-menu-button__items" role="menu">
+                            <div class="filter-menu-button__item" role="menuitemcheckbox" aria-checked="false">
+                                <span class="filter-menu-button__status--inline">
+                                    <svg class="icon icon--checkbox-unchecked" focusable="false" height="18" width="18">
+                                        <use xlink:href="#icon-checkbox-unchecked"></use>
+                                    </svg>
+                                    <svg class="icon icon--checkbox-checked" focusable="false" height="18" width="18">
+                                        <use xlink:href="#icon-checkbox-checked"></use>
+                                    </svg>
+                                </span>
+                                <span class="filter-menu-button__text">Item 1 with a long string that will wrap to another line</span>
+                            </div>
+                            <div class="filter-menu-button__item" role="menuitemcheckbox" aria-checked="false">
+                                <span class="filter-menu-button__status--inline">
```

Comment:
@seangates @msendlakowski so this got merged with the invalid BEM still present? :-( A base class needs to be present, or this modifier needs to be converted to a base class.